app/services/config_report_importer.rb

```diff
@@ -35,3 +43,5 @@ def statuses_for_refresh
     [HostStatus.find_status_by_humanized_name("configuration")]
   end
 end
+
+ReportImporter.register_smart_proxy_feature("Puppet")
```

Comment:
Should this be in an initializer? I wonder how it would deal with code reloads.

Reply:
It should behave fine, but it is ugly

Reply:
I've fixed that manually